homework/03LessonThree.js

```diff
@@ -110,10 +110,13 @@ var threeByThreeArray = [
 
 // Problem 11:
 // Replace the center element of threeByThreeArray, which is 5, with something else.
-
+threeByThreeArray[1][1] = 9001
 // Problem 12:
 // create a 2 by 2 (2 rows, 2 columns) nested array
-var twoByTwoArray;
+var twoByTwoArray = [
+  [0, 1],
+  [2, 3]
+];
 
```

Comment:
All your tests passed. Good work!

![good job](http://www.picgifs.com/reaction-gifs/reaction-gifs/good-job-succes-nailed-it/good-job44.gif)